src/app/search/state/search.query.ts

```diff
@@ -6,31 +6,32 @@ import { map } from 'rxjs/operators';
 import { AppTool, DockstoreTool, Workflow, Notebook } from '../../shared/openapi';
 import { SearchState, SearchStore } from './search.store';
 
+export interface SearchResult<T = AppTool | DockstoreTool | Workflow | Notebook> {
+  source: T;
+  highlight: any;
```

Comment:
In my limited tests, this is always a `Map<string, string[]>`; if it always is, it would be nice to have the type, even though the ES code is already full of `any`.